lmdeploy/model.py

```diff
@@ -1013,6 +1013,32 @@ def messages2prompt(self, messages, sequence_start=True):
         return ret
 
 
+@MODELS.register_module(name=['deepseek-chat'])
+class Deepseek(BaseModel):
```

Comment:
@AllentDan Do we need to implement `messages2prompt` method of this class?

Reply:
For api_server's `v1/chat/completions`, yes.